src/Utilities/Microsoft.Build.Utilities.csproj

```diff
@@ -209,15 +209,20 @@
     </Compile>
     <!-- Win32 RC Files -->
     <RCResourceFile Include="native.rc" />
+    
```

Comment:
I specifically added them in to make the csproj easier to read. I'll take them out if I'm breaking some styling convention :)